imagetest/test_suites/networkperf/setup.go

```diff
@@ -95,12 +95,14 @@ var scripts embed.FS
 var targets embed.FS
 
 const (
```

Comment:
I realize this is both pedantic and pre-exisitng but technically these are URIs not URLs since they're on the local filesystem.

I only bring this up because the `startup-script-url` is the thing that we to execute the test wrapper and it's slightly confusing to reference the script on the filesystem as `linuxInstallStartupScriptURL` but then we actually execute the `linuxInstallStartupScriptURL` as the `startup-script` and execute the wrapper as the `startup-script-url`. This confused me at first when I read the section lower down on reading `linuxInstallStartupScriptURL` and I thought you were switching the mechanism from `startup-script` to `startup-script-url`. I think referencing these as URIs would clear up some potential confusion.

Reply:
Done.